build.sbt

```
@@ -54,12 +54,11 @@ Global / excludeLintKeys += debugSettings // To avoid lint warning
 
 mainClass in reStart := Some("org.broadinstitute.dsde.rawls.Boot")
 
-Revolver.enableDebugging(port = 5050, suspend = false)
-
 // When JAVA_OPTS are specified in the environment, they are usually meant for the application
 // itself rather than sbt, but they are not passed by default to the application, which is a forked
 // process. This passes them through to the "re-start" command, which is probably what a developer
 // would normally expect.
-sys.env.getOrElse("JAVA_OPTS", "").split(" ").toSeq.map { opt =>
-  javaOptions in reStart += opt
-}
+javaOptions in reStart ++= sys.env.getOrElse("JAVA_OPTS", "")
+  .concat(" -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5050")
+  .split(" ")
+  .toList
```

If someone can tell me how to reorder the operations so I can remove the leading space before the `-agentlib` argument, I'll be very happy.

---

```suggestion
javaOptions in reStart ++= sys.env.getOrElse("JAVA_OPTS", "")
  .split(" ")
  .toList :+ "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5050"
```

Have not confirmed that this works